docs/webhooks/20-Early-Access-Webhooks.md

```diff
@@ -111,3 +117,51 @@ Depending on the `event.event_type`, of the webhook payload, the `event.data` fi
   }
 }
 ```
+
+### incident_role_assignment
+```json
+{
+  "id": "cc7f4cc9-02be-47c9-b826-72c7976d84f3",
+  "event_type": "incident.role.assigned",
+  "resource_type": "incident",
+  "occurred_at": "2021-02-08T18:35:57Z",
+  "agent": {
+    "html_url": "https://acme.pagerduty.com/users/PDJKATD",
+    "id": "PDJKATD",
+    "self": "https://api.pagerduty.com/users/PDJKATD",
+    "summary": "User 22",
+    "type": "user_reference"
+  },
+  "client": null,
+  "data": {
```

Comment:
Since the envelope is the same for all webhooks, we just put this `data` object here in the docs in this section. Can you pare it down to just this part?